lib/views/pages/post/post_page.dart

```diff
@@ -38,17 +39,22 @@ class PostPage extends HookConsumerWidget {
       ),
     ];
 
+    final commentList = CircularValue(
+      value: commentsAsync,
+      builder: (context, comments) => CommentList(
+        key: commentListWidgetKey,
+        comments: comments,
+      ),
+    );
+
     // Body = Complete post + Comments
     List<Widget> bodyChildren = [
       CompletePostWidget(
         key: completePostWidgetKey,
         post: postOverview,
       ),
       const SizedBox(height: 10),
-      CommentList(
-        key: commentListWidgetKey,
-        comments: comments,
-      ),
+      commentList,
```

Comment:
This a very personal opinion, but I feel like creating a lot of the interface using different variables makes it less readable. Personally, I'd prefer (quite a lot) if this variable was inlined.

Reply:
Yes I think you are right.
I had done that because `CommentList` is now wrapped up with a `CircularValue` and I though it might little to much to have everything inline.
But looking again at it I think it is more readable when inlined.

Fix in e75f07f46da9b8bd15600d7737a3215b790d2aba.